app/qml/main.qml

```diff
@@ -606,6 +606,26 @@ ApplicationWindow {
       }
     }
 
+    MessageDialog {
+      id: projectErrorDialog
+
+      title: qsTr("Failed to open the project")
+      buttons: MessageDialog.Close | MessageDialog.Help
+
+      onButtonClicked: function(button, role) {
+        switch (button) {
+          case MessageDialog.Close:
+            projectLoadingScreen.visible = false
+            projectErrorDialog.close()
+            projectPanel.openPanel()
+            break;
+          case MessageDialog.Help:
+            Qt.openUrlExternally(__inputHelp.projectLoadingErrorHelpLink)
```

Comment:
We also need to close the project loading screen here. Otherwise, it remains visible and one must restart the app to get out.

Reply:
We already do it, no? Line 618.

Reply:
That is when you click `Close`, but on ios also the `Help` button closes the dialog

Reply:
Probably because you are navigated outside of the app (docs)

Reply:
Should be fixed now.